### PR TITLE
2022 extend owl2 tests

### LINKED ISSUE #2022: Check and extend OWL2 tests
Part of #1945

Adjust test cases for OWL2 to new syntax.
- change existing dol files with MS snippets
- add dol files with AS snippets
- add dol, ofn, omn files with SWRL Rules

### PR DESCRIPTION
**Pull request by b-gehrke**
Fixes #2022

### CODE REVIEW

**Comment by b-gehrke** [comment]
All tests pass. All warnings fixed.

File: OWL2/StaticAnalysis.hs [hunk 1]
```diff
@@ -90,6 +90,22 @@ filterObjProp :: Sign -> [AS.ObjectPropertyExpression]
     -> [AS.ObjectPropertyExpression]
 filterObjProp = filter . isDeclObjProp
 
+
+checkHasKey :: Sign -> [AS.ObjectPropertyExpression] -> [AS.DataPropertyExpression] -> Result ([AS.ObjectPropertyExpression], [AS.DataPropertyExpression])
```
Comment (mcodescu) [inline OWL2/StaticAnalysis.hs:94]: Shorten this line.

**Review by mcodescu — CHANGES_REQUESTED** [review]
Very minor change.

**Review by mcodescu — APPROVED** [review]
Looks good to me.
:+1:

**Pull request merged**